packages/paste-website/src/pages/inclusive-design/index.mdx

```diff
@@ -0,0 +1,409 @@
+        <ListItem>
+          Do not use very thin font faces{' '}
+          <Anchor
+            showExternal
+            href="http://www.telegraph.co.uk/science/2016/10/23/internet-is-becoming-unreadable-because-of-a-trend-towards-light/"
+          >
+            (Read an article about why this matters)
+          </Anchor>
+        </ListItem>
+        <ListItem>Underline links — at least in body copy</ListItem>
+        <ListItem>
+          Avoid justified body text since it creates rivers of spacing that can be hard to read.{' '}
+          <Anchor showExternal href="https://www.w3.org/TR/WCAG20-TECHS/F88.html">
+            (Read more about why this matters)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Provide enough spacing between lines of text (`line-height`) to allow for readers to track text{' '}
+          <Anchor showExternal href="https://www.w3.org/TR/WCAG20-TECHS/C21.html">
+            (Read more about this)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Avoid all-caps text{' '}
+          <Anchor showExternal href="https://github.com/humanmade/hm-pattern-library/issues/75">
+            (Read more about this)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Ensure that content is written as clearly and simply as possible{' '}
+          <Anchor showExternal href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-supplements.html">
+            (Read more about why it matters)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Warn users of links that have unusual behaviors, like linking off-site, or loading a new tab
+        </ListItem>
+        <ListItem>Use textual labels to make voice activation cues obvious</ListItem>
+        <ListItem>Do not mark up subheadings/straplines with separate heading elements</ListItem>
+        <ListItem>Begin long, multi-section documents with a table of contents</ListItem>
+        <ListItem>
+          Make sure the purpose of a link is clearly described: "read more" vs. "read more about accessibility"
+        </ListItem>
+      </UnorderedList>
+    </DisclosureContent>
+  </Disclosure>
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Visual design and visual affordances
+    </DisclosureHeading>
+    <DisclosureContent>
+      <UnorderedList>
+        <ListItem>Include only clear, meaningful animations</ListItem>
+        <ListItem>Do not rely on color as the only means of differentiation of visual elements</ListItem>
+        <ListItem>Make UI controls look like controls; give them strong perceived affordance</ListItem>
+        <ListItem>Avoid pure white or pure black shades</ListItem>
+        <ListItem>Indicate swipe gesture support clearly, and provide simple tap-based alternatives</ListItem>
+        <ListItem>
+          Avoid time constraints where possible; provide a clear warning and option to extend where not possible
```

Comment:
This one feels mis-categorized, I think. It might just belong under "other considerations".